submissions/SerhiiYakovenko/html-css-popup/index.html

```diff
@@ -0,0 +1,152 @@
+                class="popup-item"
+                target="_blank"
+              >
+                <img src="./img/instagram.svg" alt="instagram" />
+                <p>Instagram</p>
+              </a>
+              <a
+                href="https://www.youtube.com/user/CyberPunkGame"
+                class="popup-item"
+                target="_blank"
+              >
+                <img src="./img/youtube.svg" alt="youtube" />
+                <p>Youtube</p>
+              </a>
+              <a
+                href="http://twitter.com/cyberpunkgame"
+                class="popup-item hidden"
+                target="_blank"
+              >
+                <img src="./img/twitter.svg" alt="twitter" />
+                <p>Twitter</p>
+              </a>
+              <a
+                href="https://www.twitch.tv/cdprojektred"
+                class="popup-item"
+                target="_blank"
+              >
+                <img src="./img/twitch.svg" alt="twitch" />
+                <p>Twitch</p>
+              </a>
+              <input type="checkbox" id="more-btn" class="more-input input" />
+              <label for="more-btn" class="more-label">Explore more!</label>
+              <a
+                href="https://forums.cdprojektred.com/index.php?forums/cyberpunk.21/"
+                class="popup-item hidden"
+                target="_blank"
+              >
+                <img src="./img/forum.svg" alt="forum" />
+                <p>Forum</p>
+              </a>
+              <a
+                href="https://discord.gg/cyberpunkgame"
+                class="popup-item hidden"
+                target="_blank"
+              >
+                <img src="./img/discord.svg" alt="discord" />
+                <p>Discord</p>
+              </a>
+              <a
+                href="https://cyberpunkgame.tumblr.com/"
+                class="popup-item hidden"
+                target="_blank"
+              >
+                <img src="./img/world.svg" alt="world" />
+                <p>World</p>
+              </a>
+              <a class="kottans hidden" href="https://github.com/kottans"
+                >Even more from Kottans</a
+              >
+            </div>
+          </div>
+        </li>
+        <li class="nav-item">
+          <a
+            href="https://www.microsoft.com/p/cyberpunk-2077/BX3M8L83BBRW"
+            class="menu-item xbox"
+            tabindex="0"
+            target="_blank"
+          >
+            <img src="./img/xbox.svg" alt="xbox" />
+          </a>
+        </li>
+        <li class="nav-item">
+          <a
+            href="https://store.playstation.com/en-gb/product/EP4497-CUSA16579_00-0000000000000001"
+            class="menu-item ps4"
+            tabindex="0"
+            target="_blank"
+          >
+            <img src="./img/ps4.svg" alt="ps4" />
+          </a>
+        </li>
+      </ul>
+    </nav>
+
+    <a
```

Comment:
It seems this content plays an important role on this page. So maybe it will be better to put this link to `h1` (if it's quite an important title) or to `h2`.